<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-E6BGSPU" failures="1" tests="3" name="ui.VerifyTitleAndTextTest" time="1.676" errors="0" timestamp="2020-10-14T02:42:06 WEST" skipped="0">
  <testcase classname="ui.VerifyTitleAndTextTest" name="titleTest" time="0.010"/>
  <system-out/>
  <testcase classname="ui.VerifyTitleAndTextTest" name="aftTest" time="1.598">
    <failure message="The following asserts failed:
	The title isn&#039;t correct.... expected [0Electronics, Cars, Fashion, Collectibles &amp; More | eBay] but found [Electronics, Cars, Fashion, Collectibles &amp; More | eBay]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
The title isn't correct.... expected [0Electronics, Cars, Fashion, Collectibles & More | eBay] but found [Electronics, Cars, Fashion, Collectibles & More | eBay]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at ui.VerifyTitleAndTextTest.aftTest(VerifyTitleAndTextTest.java:46)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:617)
at org.testng.TestRunner.afterRun(TestRunner.java:847)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- aftTest -->
  <system-out/>
  <testcase classname="ui.VerifyTitleAndTextTest" name="buttonTest" time="0.068"/>
  <system-out/>
</testsuite> <!-- ui.VerifyTitleAndTextTest -->
